model/error/event_test.go

```diff
@@ -791,13 +809,16 @@ func TestCulprit(t *testing.T) {
 		},
 	}
 	for idx, test := range tests {
-		tctx := &transform.Context{
-			Config: test.config,
-		}
+		t.Run(string(idx), func(t *testing.T) {
```

Comment:
`string(idx)` is going to create strings with runes '\u0000', '\u0001', etc. I guess you meant `fmt.Sprint(idx)`?